### DISCUSSION TITLE
Agenda for the Flatcar Release Team Call on Tuesday, 25th of January 2022 / 5:30pm CET / 10pm IST / 4:30pm GMT / 11:30am EST / 8:30am PST

### DISCUSSION THREAD

**Discussion by sayanchowdhury** [flatcar/Flatcar#601] [Flatcar Developer Sync]
## Agenda for the Flatcar Release Team Call on Tuesday, 25th of January 2022 / 5:30pm CET / 10pm IST / 4:30pm GMT / 11:30am EST / 8:30am PST

* Zoom link: [https://us06web.zoom.us/j/82054240491](https://us06web.zoom.us/j/82054240491)
  * Meeting ID: 820 5424 0491
  * Passcode: 444888
- Youtube live stream / recording: https://www.youtube.com/watch?v=Wy4PpImxn5k
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## News
- Brief introduction about the upcoming FOSDEM devroom talks overview
- Talk about the upcoming Bug-Smashing day.

## Release planning
- Ongoing and upcoming releases (https://github.com/orgs/flatcar-linux/projects/5)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by t-lo** [comment]
Last-minute news items we might cover:
- release process if fully public, w/ tracking issues of release todos etc.
- Flatcar maintainers team will be holding a bug smashing day on Friday, January 28th! Join us to make your voice heard on your favourite nuisance, or tune in and help us smashing bugs from our issue tracker! Find all the details here: https://github.com/flatcar-linux/Flatcar/issues/603
